### PR TITLE
Added segmented-colormap Module

### PR DESCRIPTION
**Pull request by ccpandhare**
## Usage
```js
sequencer.loadImage('PATH').addSteps('segmented-colormap',options).run()
```
## Where
`options.colormap` may be either of "greyscale","default","stretched","fastie".
Default value of `options.colormap` is "default"

## Example
```js
sequencer.loadImage('examples/replace.jpg')
         .addSteps('segmented-colormap',{colormap:"fastie"})
         .run()
```

## Confirmed from Infragram
I have visually confirmed that the output of the module is similar to that of [Infragram](https://infragram.org), if not the same.

### CODE REVIEW

File: src/modules/DoNothingPix.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 /*
  * Display only the green channel
```
Comment (jywarren) [inline src/modules/DoNothingPix.js]: oops looks like this needs a change

**Comment by jywarren** [comment]
This looks super -- is there a way to specify a colormap on the fly by passing in the nested array of values? Maybe with param `colormap` just being an array?

**Comment by ccpandhare** [comment]
> is there a way to specify a colormap on the fly by passing in the nested array of values? Maybe with param colormap just being an array?

I have added this now!

**Comment by ccpandhare** [comment]
## Updates
* description of DoNothingPix updated.
* added support for custom Colormaps

**Comment by jywarren** [comment]
Looks great, merge away! Let's add this to the `docs/modules.md` documentation too.

**Comment by ccpandhare** [comment]
Sure!
Merging!

**Pull request merged**